cli/tests/unit/chmod_test.ts

```diff
@@ -18,6 +18,25 @@ unitTest(
   }
 );
 
+unitTest(
+  { ignore: Deno.build.os === "windows", perms: { read: true, write: true } },
+  function chmodSyncUrl(): void {
+    const enc = new TextEncoder();
+    const data = enc.encode("Hello");
+    const tempDir = Deno.makeTempDirSync();
+    const fileUrl = new URL(`file://${tempDir}/test.txt`);
```

Comment:
This is not a correct file URL. Correct file URLs always start with "file:///" except maybe when you're dealing with a network path (`\\server\share\dir\file`) which is a windows only thing that we don't need to support very urgently.

Reply:
nvm - missed that this is a posix-only test